Automattic/jetpack
# Update translations from the repository

Fixes #6780 

#### Changes proposed in this Pull Request:
* Updated existing translations for client side l10n.
* Updated existing translation for server side l10n.
* Updated translation texts and binaries for languages that have no language packs yet, but have a certain threshold of translated texts.
* Added translation for languages that have not yet come over the threshold, but still have a considerable amount of translated texts.

#### New added languages
* Amharic
* Assamese
* South Azerbaijani
* Belarusian
* Cebuano
* English (South Africa)
* Spanish (Mexico)
* Estonian
* Basque
* French (Canada)
* Gujarati
* Hindi
* Armenian
* Luxembourgish
* Marathi
* Occitan
* Punjabi
* Sindhi

### Updated existing languages with significant translation counts
These languages have no translation packs, so we supply .mo files with the plugin. These languages have high translation counts and are updated with new translations from the repository.
* Afrikaans
* Bulgarian
* Czech
* Welsh
* Hungarian
* Icelandic
* Georgian
* Latvian
* Malay
* Norwegian (Bokmål)
* Polish
* Telugu
* Thai
* Urdu
* Vietnamese

### Updated stale languages 
These languages have not had an increase in translation counts but have had .mo files that weren't updated for a while, so we download and update them as well.
* Moroccan Arabic
* Azerbaijani
* Bosnian
* Catalan
* Kurdish (Sorani)
* Danish
* Scottish Gaelic
* Galician
* Lithuanian
* Macedonian
* Norwegian (Nynorsk)
* Portuguese (Portugal)
* Sinhala
* Serbian
* Ukrainian
* Chinese (Hong Kong)

#### Testing instructions:
* Pick any language you know from the set and try to use it on your site. Depending on the translation level you will see at least some translated phrases. You can check what to look for [in the translation interface](https://translate.wordpress.org/projects/wp-plugins/jetpack/dev). Make sure you can see phrases translated in the new Settings interface as well - all languages have their JSON files updated.
